tool/speed.cc

```diff
@@ -2425,8 +2425,8 @@ bool Speed(const std::vector<std::string> &args) {
        !SpeedHash(EVP_sha256(), "SHA-256", selected) ||
        !SpeedHash(EVP_sha384(), "SHA-384", selected) ||
        !SpeedHash(EVP_sha512(), "SHA-512", selected) ||
-       // OpenSSL 1.0 doesn't support SHA3.
-#if !defined(OPENSSL_1_0_BENCHMARK) && AWSLC_API_VERSION > 16
+       // OpenSSL 1.0 and BoringSSL doesn't support SHA3.
+#if !defined(OPENSSL_1_0_BENCHMARK) && !defined(BORINGSSL_BENCHMARK) || AWSLC_API_VERSION > 16
```

Comment:
Only noticed this after approving: Is this supposed to be `&& AWSLC_API_VERSION > 16`, or is `||` intentional

Reply:
It's supposed to be `||`, since AWS-LC is the only one to define `AWSLC_API_VERSION`, for BoringSSL and OpenSSL 1.1.1/3.0 it will evaluate to false.

Reply:
I believe the precedence would evaluate `&&` before `||`, but maybe brackets around the first 2 checks will make it more readable.